### ISSUE TITLE
Closing preference window using the "x"

### ISSUE THREAD

**Issue by BedHead911** [xflux-gui/fluxgui#26]
I'm using Linux Mint 18 with the MATE desktop environment. After pressing the "x" button on the top right of the window to close the preferences window, you can no longer re-open the preference window.

![screenshot at 2016-08-20 15-54-45](https://cloud.githubusercontent.com/assets/18540453/17834293/8ad0e662-66ee-11e6-8306-775b0a713459.png)

![screenshot at 2016-08-20 15-55-28](https://cloud.githubusercontent.com/assets/18540453/17834296/93ce3c1a-66ee-11e6-906f-ff1e4b2ced49.png)

**Comment by BedHead911** [comment]
In the second picture, you can see the preferences window where my mouse cursor is... all the contents are missing.

**Comment by serkazak** [comment]
I replicated this problem on debian with cinnamon.

**Comment by itsabear** [comment]
Actually, this is happening on my machine *Ubuntu 16.04*

```
Linux pc 4.4.0-36-generic #55-Ubuntu SMP Thu Aug 11 18:00:59 UTC 2016 i686 i686 i686 GNU/Linux
```

**Comment by theblackhole** [comment]
Same here on Linux Mint 18.

If I start fluxgui via the terminal I get these warning messages. I don't know if it's useful :

```
/usr/lib/python2.7/dist-packages/fluxgui/fluxapp.py:75: GtkWarning: Cannot transform xsetting Net/IconThemeName of type gchararray to type GdkColor

  gtk.gdk.screen_get_default().get_setting('gtk-icon-theme-name')
/usr/lib/python2.7/dist-packages/fluxgui/fluxapp.py:75: GtkWarning: Cannot transform xsetting Net/IconThemeName of type gchararray to type gint

  gtk.gdk.screen_get_default().get_setting('gtk-icon-theme-name')
```
 
_fluxgui 1.1.9~20160808-g4125e70-1~xenial
Linux Mint 18 Cinnamon x64_

**Comment by ntc2** [comment]
Going to assume this is fixed by #112. Please let me know if it's not.
